apps/ubuntu_bootstrap/lib/app/installation_step.dart

```diff
@@ -106,6 +110,9 @@ enum InstallationStep with RouteName {
         }
         return false;
       },
+      onBack: this == InstallationStep.autoinstallLandscapeError
+          ? (_) => InstallationStep.autoinstall.route
+          : null,
```

Comment:
Could this be a property on the `InstallationStep` enum that is `null` for every step but this one?